snmp/files/snmpd.conf

```diff
@@ -6,7 +6,7 @@
 ###############################################################################
 #
```

Comment:
Mind adding a note that says this file is managed by SaltStack while you're in here?